test/f90_correct/src/format_specification.f90

```diff
@@ -0,0 +1,28 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! Checking for the token type "TK_G0FORMAT" (I0) and
+! the special case of the token type "I" + "<integer>" (I02),
+! while the normal case is I2.
+
+program example
```

Comment:
There are complains in other reviews for giving the programs generic names like `test` or `example`. Personally, I'm not against, but others may ask you to change it.

Reply:
I think it's modules with non-unique names that could cause problems.